style.css

```diff
@@ -10,6 +10,8 @@
   }
   
   #image {
+    /* display: flex;
+    margin-top: 0cap; */
```

Comment:
- Everything is perfect in your css code , good job!!. but i would suggest that you should not keep unused code or comments in your codebase